src/Microsoft.DotNet.Tools.Init/Template/project.json

```diff
@@ -6,7 +6,7 @@
 
     "dependencies": {
         "Microsoft.NETCore.Runtime": "1.0.1-beta-*",
-        "System.IO": "4.0.10-beta-*",
+        "System.IO": "4.0.11-beta-*",
```

Comment:
I'm getting lost on those - first we recommended lowest possible version so that we can target as much as we can and now updating to highest :) We should just have a "*" (give me newest or whatever is the best) version